packages/package_stringmlst_1_0/tool_dependencies.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="stringmlst" version="1.0">
+        <install version="1.0">
+          <actions>
+            <action type="download_by_url" sha256sum="164f1ea5f486036dbb77481d0cc560315f40ce7aecffbb0f28521431be188de0">https://github.com/anujg1991/stringMLST/archive/v0.2.1.tar.gz</action>
+            <action type="shell_command">cp ./stringMLST.py $INSTALL_DIR</action>
```

Comment:
should use the move_file tag as documented [here](https://wiki.galaxyproject.org/ToolDependenciesTagSets).